ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/button/_cq_dialog/.content.xml

```diff
@@ -0,0 +1,77 @@
+                                        sling:resourceType="granite/ui/components/coral/foundation/container">
+                                        <items 
+											jcr:primaryType="nt:unstructured"
+											sling:hideChildren="icon">
+                                            <buttonType
+                                                jcr:primaryType="nt:unstructured"
+                                                sling:orderBefore="text"
+                                                sling:resourceType="granite/ui/components/coral/foundation/form/select"
+                                                fieldLabel="Button Type"
+                                                name="./buttonType">
+                                                <items jcr:primaryType="nt:unstructured">
+                                                    <primary
+                                                        jcr:primaryType="nt:unstructured"
+                                                        text="Primary"
+                                                        value="btn-primary"/>
+                                                    <secondary
+                                                        jcr:primaryType="nt:unstructured"
+                                                        text="Secondary"
+                                                        value="btn-secondary"/>
+													<ternary
+                                                        jcr:primaryType="nt:unstructured"
+                                                        text="Ternary"
+                                                        value="btn-ternary"/>
+                                                    <link-primary
+                                                        jcr:primaryType="nt:unstructured"
+                                                        text="Link"
+                                                        value="link-primary"/>
+                                                </items>
+                                            </buttonType>
+											<disabledButton
+                                                jcr:primaryType="nt:unstructured"
+                                                sling:resourceType="granite/ui/components/coral/foundation/form/checkbox"
+                                                fieldDescription="Checkbox to disable the button"
+                                                name="./disabledButton"
+												text="Disable Button"
+												value="__disabled"/>
+                                        </items>
+                                    </column>
+                                </items>
+                            </columns>
+                        </items>
+                    </properties>
+                </items>
+            </tabs>
+        </items>
+    </content>
+</jcr:root>
```

Comment:
@rakeshkumar-verma : put back the icon deialog field.

Reply:
done